docs/Parameters.md

```diff
@@ -54,8 +54,8 @@ The parameter format is `key1=value1 key2=value2 ... ` . And parameters can be s
   * support multi validation data, separate by `,`
 * `num_iterations`, default=`100`, type=int, alias=`num_iteration`,`num_tree`,`num_trees`,`num_round`,`num_rounds`
   * number of boosting iterations
-  * note: `num_tree` here equal with `num_iterations`. For multi-class, it actually learns `num_class * num_iterations` trees.
-  * note: For python/R package, cannot use this parameters to control number of iterations.
+  * note: For python/R package, **this parameters is ignored**.
```

Comment:
@i3v your proposed note seems easier to understand and explicit

The reason you can still pass the number of iterations in parameters and it is ignored is because the API requires to do per-iteration boosting, and not a whole number of iterations of boosting. This is by design (so we can have callbacks / custom losses/metrics / watchlists). Otherwise, it would be simpler to use a CLI wrapper in R/Python than the current wrapper, but you would have no access to any callbacks and training control.